lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -122,23 +84,37 @@ def endpoint
 
     def fulfill(order_id, shipping_address, line_items, options = {})
       requires!(options, :order_date, :shipping_method)
-      commit :post, :outbound, :create, build_fulfillment_request(order_id, shipping_address, line_items, options)
+      with_error_handling do
+        data = commit :post, 'CreateFulfillmentOrder', build_fulfillment_request(order_id, shipping_address, line_items, options)
+        parse_fulfillment_response(parse_document(data), 'Successfully submitted the order')
+      end
     end
 
     def status
-      commit :post, :outbound, :status, build_status_request
+      with_error_handling do
+        data = commit :post, 'GetServiceStatus', build_basic_api_query({ :Action => 'GetServiceStatus' })
+        parse_tracking_response(parse_document(data))
+      end
```

Comment:
@wvanbergen I *think* this is doing the same thing as before. The `commit` method ended with the same `rescue ActiveUtils::ResponseError => e` that I have in `with_error_handling`.